flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_WindPage.js

```diff
@@ -248,4 +287,151 @@ class CDUWindPage {
             speed: speed
         };
     }
+
+    static WindRequest(mcdu, stage, _showPage) {
+        getSimBriefOfp(mcdu, () => {}, () => {
+            const windData = [];
+            const lastAltitude = 0;
```

Comment:
You are assigning to these variables later even though they're marked as `const`. Did this work for you?
```suggestion
            let windData = [];
            let lastAltitude = 0;
```

Reply:
The `const` array is expected as it's never reassigned, only `push`ed ongo, but I'm surprised both me and the linter missed the reassignment on `lastAltitude`.